telemetry/definitions/commonDefinitions.json

```diff
@@ -15,6 +15,16 @@
             "type": "double",
             "description": "Total time from start to end"
         },
+        {
+            "name": "amazonqFileExtensionName",
```

Comment:
is there a reason the existing filenameExt field can't be used? https://github.com/aws/aws-toolkit-common/blob/7b972e3fe7ee01b3b3d99c4198915c1520c0401e/telemetry/definitions/commonDefinitions.json#L1129

please check for existing fields instead of adding new ones. same for the `amazonQFileIgnoredCount` field.

Reply:
no reason, let me adjust, will keep the `amazonQFileIgnoredCount` as it has a different use case/meaning